utilities/logic/include_vars.py

```diff
@@ -10,28 +10,47 @@
 
 DOCUMENTATION = '''
 ---
-author: "Benno Joy (@bennojoy)"
+author: "Allen Sanabria (@linuxdynasty)"
 module: include_vars
 short_description: Load variables from files, dynamically within a task.
 description:
-     - Loads variables from a YAML/JSON file dynamically during task runtime.  It can work with conditionals, or use host specific variables to determine the path name to load from.
+     - Loads variables from a YAML/JSON files dynamically from within a file or
```

Comment:
I would also lose the comma after the word 'files' in the short description. Other than that it looks good to go. Thanks!